ros_gz_sim_demos/config/triggered_camera.yaml

```
@@ -0,0 +1,27 @@
+# triggered_cammera configuration.
+- ros_topic_name: "/camera"
+  gz_topic_name: "/camera"
```

This can be simlpified
```suggestion
- topic_name: "/camera"
```

---

Simplified in [3c5d49b](https://github.com/gazebosim/ros_gz/pull/645/commits/3c5d49b471c9a569691f3ffcf052ee437e23808d).